evento/app/controllers/users_controller.rb

```diff
@@ -63,7 +64,11 @@ def set_user
   end
 
   # Only allow a trusted parameter "white list" through.
-  def user_params
-    params.require(:user).permit(:name, :email, :password)
+  def create_params
```

Comment:
Interesting. Want to fix that? As a side note, does that mean that our controllers at the moment allow non-json requests? Should we somewhow block them?